server.go

```diff
@@ -753,24 +753,33 @@ func (w *response) Write(m []byte) (int, error) {
 
 		n, err := io.Copy(w.tcp, bytes.NewReader(m))
 		return int(n), err
+	default:
+		panic("Write called after Close")
```

Comment:
for panic it's customary to prefix it them with the package: `panic("dns: write called after Close")`

Reply:
I don't know how I missed that, I always prefix errors and panics in my own packages. :man_facepalming: